sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -192,20 +228,16 @@ private JdbcIO() {}
     }
 
     public static DataSourceConfiguration create(DataSource dataSource) {
-      checkArgument(dataSource != null, "DataSourceConfiguration.create(dataSource) called with "
-          + "null data source");
-      checkArgument(dataSource instanceof Serializable,
-          "DataSourceConfiguration.create(dataSource) called with a dataSource not Serializable");
+      checkArgument(dataSource != null, "dataSource can not be null");
```

Comment:
Why not keeping the previous messages to be aligned with other IOs ?

Reply:
The style of messages is not really consistent between all IOs and I'm changing this one to be according to the best practices recently discussed on dev@ and currently recommended by PT Style Guide https://beam.apache.org/contribute/ptransform-style-guide/#validation